File: app/src/main/java/co/tpcreative/supersafe/common/response/VerifyCodeResponse.java
package co.tpcreative.supersafe.common.response;

import co.tpcreative.supersafe.common.api.response.BaseResponse;
import co.tpcreative.supersafe.model.User;

public class VerifyCodeResponse extends BaseResponse{
    public String code ;
    public User user;
}

File: app/src/main/java/co/tpcreative/supersafe/common/response/VerifyCodeResponse.kt
package co.tpcreative.supersafe.common.response
import co.tpcreative.supersafe.common.api.response.BaseResponse
import co.tpcreative.supersafe.model.User

class VerifyCodeResponse : BaseResponse() {
    var code: String? = null
    var user: User? = null
}